client/src/components/LikeList/index.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import '../../styles/style.scss';
+import '../Post/style.scss';
+import noAvatar from '../../assets/noAvatar.png';
+import { Link } from 'react-router-dom';
+import { formatDescription } from '../../helpers/regex.description';
+import FeedLikesContainer from '../../containers/FeedLikesContainer';
+import Menu from '../Menu';
+import { Col, Container, Row, Spinner } from 'reactstrap';
+import { Waypoint } from 'react-waypoint';
+import Comment from '../Comments';
+import WriteComment from '../WriteComment';
+import { INewsFeed } from '../../store/likesList/reducers';
+
+interface IProps {
+    likeList: any;
```

Comment:
I believe in @yuriimartynenko PR , I saw type string[]

Reply:
fixed (changed to Interface)